commontype/fee_config.go

```diff
@@ -144,6 +146,65 @@ func (f *FeeConfig) checkByteLens() error {
 	return nil
 }
 
+func (c *FeeConfig) getBigIntToSerialize() []**big.Int {
+	return []**big.Int{
+		&c.GasLimit, &c.MinBaseFee, &c.TargetGas, &c.BaseFeeChangeDenominator,
+		&c.MinBlockGasCost, &c.MaxBlockGasCost, &c.BlockGasCostStep,
+	}
+}
+
+func (c *FeeConfig) MarshalBinary() ([]byte, error) {
```

Comment:
Can we add the following test cases for the serialization of `FeeConfig`:

1. serialize and de-serialize valid config and get the same value
2. hardcoded test vector and require it results in the same value
3. unmarshal should fail if there are insufficient bytes to unpack big int values